implementation_guide.md

```diff
@@ -0,0 +1,95 @@
+# Implementation Strategy
+
+## Git Subtrees
+
+The crux of symbiflow-common-config is git subtrees. This is a tool built right into git, allowing you to nest one repository inside of another as a dependency.
+
+Subtrees offer numerous advantages over other commonly used dependency tools like git submodules (easy setup and maintenance, no new commands that developers must learn in order to work with the dependencies, etc.)
+
+In our case, symbiflow-common-config is the repository that will be merged into all other SymbiFlow projects as a git subtree.
+
+To learn more about git subtrees and how they work, see [this page](https://www.atlassian.com/git/tutorials/git-subtree).
+
+
+## How files will be organized in symbiflow-common-config
+
+While we already have a draft of many of the files we hope to include in symbiflow-common-config, we will not be implementing them all at once.
+
+To accommodate this, we will push groups of similar files to branches other than “main”, granting the option to merge in any of these feature sets to test as we see fit.
+
+<img src="assets/img/branches-example.png" width=300px> 
+
+## Testing strategy
+
+As said earlier, we are using a wide approach, rather than deep. That is to say, we will implement one small feature organization-wide before moving on to other features.
+
+The first feature we will test/implement is the default community health files (`CONTRIBUTING.md`, `CODE_OF_CONDUCT.md`, etc.).
+
+Adding these files would not significantly alter/change the functionality of repositories, but they are still important and missing in many SymbiFlow projects.
+
+In the testing phase we will add the feature to a small subset of repositories, to ensure stability and have a chance to catch any unforeseen problems.
+
+The first repository we plan to add files to as a test is [`pjxray-bram-patch`](https://github.com/SymbiFlow/prjxray-bram-patch), as it does not contain many of the features common-config will implement.
+
+The other testing repositories are not yet decided, but will contain at least one that already includes common-config features, giving us the opportunity to ensure our implementation resolves these types of conflicts okay.
+
+This process will then be repeated with all other feature sets.
+
+Order of feature implementation: (organized from least to most complex)
+1. Community Health Files
+2. Issue and Pull Request Templates
+3. EditorConfig and Autoformatters
+4. Sphinx and Readthedocs setup
+
```

Comment:
Rather than say "we will" I believe it might be better to just say, "the approach is this, and that". Also I believe that this section might be reduced to include only information regarding the general strategy rather than the specific files/repos that will be integrated with the common-config first.

Thinking forward, it might be good having an issue to keep track of the common config integration process within the various repos in the symbiflow org.

Reply:
Ok, I'd say than that the starting point can be written down here (bram-patch and the fact that the initial updates will regard the community health files), and a progress issue opened on this repository as well